2-mandatory/1-writers.js

```diff
@@ -70,8 +75,17 @@ Exercise 2:
   "Writer {firstName} {lastName} died at {age} years old."
 */
 
-function logDeadWritersInTheirForties() {
+function logDeadWritersInTheirForties(array) {
   // write your code here
+  array
+    .filter(
+      (person) => person.age >= 40 && person.age < 50 && person.alive === false
+    )
+    .forEach((item) =>
+      console.log(
+        `Writer ${item["firstName"]} ${item["lastName"]} died at ${item["age"]} years old.`
+      )
+    );
```

Comment:
This will work but you could avoid iterating over the array twice by putting the check you have in the filter in an `if` condition in the foreach eg:
```
writers.forEach((person) => {
    if (person.age >= 40 && person.age < 50 && person.alive) {
      console.log(
        `Writer ${writer.firstName} ${writer.lastName} died at ${writer.age} years old.`
      );
    }
  })
```